demos/notebooks/demo_pipeline.ipynb

```diff
@@ -275,9 +380,9 @@
     "#%% compare with original movie\n",
     "display_movie = False\n",
     "if display_movie:\n",
-    "    m_orig = cm.load_movie_chain(fnames)\n",
+    "    movie_orig = cm.load_movie_chain(fnames)\n",
```

Comment:
We should have done this a long time ago. m_orig was never a great name. Glad to see this change.